src/scss/shared.scss

```diff
@@ -0,0 +1,6 @@
+// Background colors
```

Comment:
I like the name but maybe naming this file `global.scss` would make more sense. Let me know what you think!